harness/determined/pytorch/dsat/_dsat_search_method.py

```diff
@@ -1059,6 +1066,8 @@ def get_trial_list_after_successful_run(
         self,
         last_trial: DSATTrial,
     ) -> List[DSATTrial]:
+        if last_trial.search_data is None:
```

Comment:
last_trial.search_data is None

Reply:
What do you mean? What's wrong with this?